src/current/_includes/releases/v23.1/v23.1.0-alpha.1.md

```diff
@@ -180,7 +180,6 @@ Release Date: December 19, 2022
 - The **Transaction resource usage** card on the **Transaction Fingerprin**t page now includes an "Idle latency" row, representing the time spent by the application performing other work while holding this transaction open. [#92951][#92951]
 - The **Databases table** page now displays all the grants in a single row per user. [#92871][#92871]
 - Added a goroutine scheduling latency graph to the **Overload** dashboard in the DB Console. It shows what the per-node p99 scheduling latency is for goroutines. [#93217][#93217]
-- Added a top-level dropdown in the DB Console which lists tenants the user has logged into. If the cluster is not a multi-tenant host, the dropdown is not displayed. [#92694][#92694]
 - CockroachDB now prevents polling `/settings`, `/nodes_ui,` and `/cluster` endpoints on incorrect login. [#93211][#93211]
 - The **Statement** and **Transaction** pages for tenant clusters gained region columns and filters for multi-region tenants. [#92357][#92357]
```

Comment:
Should this one also be removed?

Reply:
Yeah let me do a pass for "tenant" too. Good catch.